docs/c_cpp_properties.json

```diff
@@ -0,0 +1,113 @@
+{
+    "configurations": [
+        {
+            "name": "N64 oot-gc-us",
```

Comment:
N64 GC? :thonk:

Reply:
"N64" as in "cross compiling" or "Not Windows/Linux x86 PC"
any suggestion?

Reply:
I would drop the N64 part entirely
```suggestion
            "name": "oot-gc-us",
```

Reply:
fwiw I agree-- I think its fair that the "building for n64" part is implied with this project. For anyone using decomp for anything else, they can change the names